node_modules/col-whiteboards/lib/api.js

```diff
@@ -755,7 +756,24 @@ var whiteboardAddActivity = function(socket, elements, callback) {
       AnalyticsAPI.track(socket.ctx.user, 'Add whiteboard element', AnalyticsAPI.getWhiteboardElementProperties(element, socket.whiteboard, elements.length));
     });
 
-    return callback();
+    async.each(elements, function(element, done) {
```

Comment:
@nicolaasmatthijs Thanks, would `eachSeries` be the proper way to do it serially?